src/core/stream/index.ts

```diff
@@ -449,6 +478,7 @@ export default function Stream({
       loadedEvent$,
       buffers$,
       emeManager$,
+      expiredManifestUpdate$,
```

Comment:
So we'll have both the regular refresh and the one from minimumUpdatePeriod?
Is that what we want?

Reply:
I looked into guidelines. It seems that when the MPD describes a Segment Timeline, and the minimum Update Period is > 0, the client still may need to refresh the MPD when playback comes at end of current timeline. So we should keep both logics.